docker/data-science/notebooks/ch07/2.semantic-search.ipynb

```diff
@@ -634,58 +603,68 @@
      "output_type": "stream",
      "text": [
       "bbq: {'category': 'Barbeque', 'term_vector': 'bbq^0.9191 ribs^0.6186 pork^0.5991'}\n",
-      "korean bbq: {'category': 'Korean', 'term_vector': 'bbq^0.9052 korean^0.8641 pork^0.6079'}\n",
+      "korean bbq: {'category': 'Korean', 'term_vector': 'korean^0.8095 bbq^0.7241 banchan^0.584'}\n",
```

Comment:
Note: these results are now different than what's in the mauscript. Something has changed... need to confirm what.